core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -1697,6 +1706,8 @@ abstract class RDD[T: ClassTag](
  */
 object RDD {
 
+  private[spark] val CHECKPOINT_ALL_MARKED = "spark.checkpoint.checkpointAllMarked"
```

Comment:
This sounds a little awkward. I would call this (and other similar variables) `CHECKPOINT_ALL_MARKED_ANCESTORS`